src/components/main/main.jsx

```diff
@@ -268,4 +142,9 @@ const Main = ({offersCount}) => {
   );
 };
 
+Main.propTypes = {
+  offersCount: PropTypes.number.isRequired,
+  offersList: PropTypes.arrayOf(PropTypes.string),
```

Comment:
Если не случайно не передадите значение, поламается на строке 95
```suggestion
  offersList: PropTypes.arrayOf(PropTypes.string).isRequired,
```

Reply:
Мда, в раздумьях об обязательности, видимо, упустила это.